scripts/policy/frameworks/telemetry/log.zeek

```diff
@@ -0,0 +1,190 @@
+##! Implementation of a telemetry.log and telemetry_histogram.log file
+##! using metrics accessible via the Telemetry module.
+
+@load base/frameworks/telemetry
+
+module Telemetry;
+
+export {
+	redef enum Log::ID += { LOG, LOG_HISTOGRAM };
+
+	type Info: record {
+		## Timestamp of reporting.
+		ts: time &log;
+
+		## Peer that generated this log.
+		peer: string &log;
+
+		## Contains the values "counter" or "gauge" depending on
+		## the underlying metric type.
```

Comment:
```suggestion
		## The underlying metric type.
```

Reply:
I'm not quite following this one as it's part of the line above, so leaving it.